README.md

```diff
@@ -27,17 +27,17 @@ This project contains an [IRIDA][] plugin for a pipeline to perform AMR detectio
 
 Please download the provided `amr-detection-[version].jar` from the [releases][] page and copy to your `/etc/irida/plugins` directory.  Now you may start IRIDA and you should see the pipeline appear in your list of pipelines.
 
-*Note:* This plugin requires you to be running IRIDA version >= `21.01`. Please see the [IRIDA][] documentation for more details.
+*Note:* This plugin requires you to be running IRIDA version >= `22.09.4`. Please see the [IRIDA][] documentation for more details.
```

Comment:
This is actually not correct. You still only need `21.01` IRIDA or greater. The value `22.09.4` you had set in the `pom.xml` file is only used for compile-time (not run-time).

Reply:
Woops. Reverted in b612dd8